CityPuzzle/Classes/Sql.cs

```diff
@@ -112,5 +113,23 @@ public static String ReadPuzzles()
 
             }
         }
+        public static void SaveRoom(Room room)
+        {
+            // Func<List <Puzzle>, string> convert =
+            var numbers = room.Tasks.Select(x => x.ID);
+            string converted=string.Join("", numbers+"-");
+            using (SqlConnection conn = new SqlConnection(ConnStr))
+            {
+                conn.Open();
+                var command = new SqlCommand("INSERT INTO Rooms (ID,Owner,RoomSize,Tasks) VALUES (@ID,@Owner,@RoomSize,@Tasks)", conn);
```

Comment:
Mismatch on command parametres given and parametres being set. Tasks and Latitude.
Also missing Room Participants